src/pages/saksbehandling/revurdering/bunnknapper/RevurderingBunnknapper.tsx

```diff
@@ -0,0 +1,45 @@
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
+import React from 'react';
+
+import { useI18n } from '~lib/hooks';
+
+import sharedI18n from '../../../saksbehandling/steg/sharedI18n-nb';
+
+import styles from './revurderingBunnknapper.module.less';
+
+export const RevurderingBunnknapper = (props: {
+    onTilbakeClick(): void;
```

Comment:
Det ser ut til at i alle tilfellene så gjøres det en `history.push` i disse callbackene. Da bør vi heller ta inn en streng og bruke ei lenke i stedet for en knapp.